kod/util/settings.kod

```diff
@@ -343,5 +349,9 @@ messages:
       return pbAlwaysCheckMonsterChasers;
    }
 
+   GetBroadcastManaCost()
```

Comment:
Since this is a percentage, a better name is GetBroadcastManaPercent.

Reply:
done